src/network.js

```diff
@@ -83,6 +86,17 @@ export class BlockstackNetwork {
     return this.coerceAddress('1111111111111111111114oLvT2')
   }
 
+  getMagicBytes(): string {
```

Comment:
We don't really use getters/setters elsewhere in this object -- so I think it'd be preferable to just use the MAGIC_BYTES field directly.